WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -248,7 +248,9 @@ import WordPressFlux
         didSetupView = true
 
         if readerTopic != nil {
-            configureControllerForTopic()
+            // Do not perform a sync since a sync will be executed in viewWillAppear anyway. This
+            // prevents a possible internet connection error being shown twice.
+            configureControllerForTopic(synchronize: false)
```

Comment:
I _feel_ like the number of errors we might display shouldn't dictate the underlying functionality—the UI should be flexible enough to deal with multiple, quick network calls.

Reply:
That's true but wouldn't it still be better to do 1 network call instead of 2? Honestly, this was the quickest way I could find to keep the Notices appearing twice.

Reply:
> That's true but wouldn't it still be better to do 1 network call instead of 2?

Probably? Takes a deeper understanding of this view controller than I have, so I err on the side of not changing what doesn't need to change 😜 

> this was the quickest way I could find

Sounds accurate. I'm trying to find the _best_ way